source/extensions/filters/http/adaptive_concurrency/concurrency_controller/gradient_controller.cc

```diff
@@ -114,7 +114,7 @@ std::chrono::milliseconds GradientController::applyJitter(std::chrono::milliseco
     return interval;
   }
 
-  const uint32_t jitter_range_ms = interval.count() * jitter_pct;
+  const uint32_t jitter_range_ms = interval.count() * jitter_pct / 100;
```

Comment:
Can you potentially do the normalizing of percentages to 0.0-1.0 the place where you read the jitter value from and comment there? It's a little confusing to keep track of this. Same elsewhere.